gnssanalysis/gn_io/sinex.py

```diff
@@ -25,17 +25,21 @@
 
 
 def _get_snx_header(path_or_bytes):
+    # Helper for error logging
+    def file_desc(pb) -> str:
+        return "Source file: unknown (passed as bytes)" if isinstance(path_or_bytes, bytes) else "Source file: "+str(path_or_bytes)
+    
     # read line that starts with %=SNX and parse it into dict
     snx_bytes = _gn_io.common.path2bytes(path_or_bytes)
     header_begin = snx_bytes.find(b"%=SNX")
+    if (header_begin == -1):
+        raise ValueError("Sinex header missing!", file_desc(path_or_bytes))
     header_end = snx_bytes.find(b"\n", header_begin)
+    if header_end == -1:
+        raise ValueError("Sinex header found but wasn't newline terminated (end unclear)", file_desc(path_or_bytes))
```

Comment:
I've tried to end up here but wasn't successful in editing an SNX to do so - do you have an example file where you were able to?